fonttools/smufl2sym-in-trans.jsn

```diff
@@ -0,0 +1,341 @@
+{
+  "accSagittal11LargeDiesisDown": null,
```

Comment:
What is the purpose if this file? Why .jsn and not .json?

Reply:
As I documented in the commit message, this file contains all keys for which a QT_TRANSLATE_NOOP is to be generated.

Reply:
And .jsn is because I grew up on MS-DOS with 8.3 filenames and prefer them… but if this is a sticking point, I can rename it.

Reply:
> As I documented in the commit message, this file contains all keys for which a QT_TRANSLATE_NOOP is to be generated.

Ah, sorry missed this reply... something to be very careful about, we'd need to remember adding those here **and** modifying in sym.cpp, or add just here **and** run the script, rather than adding **just** modifing in sym.cpp (as later run the script would revert that change).
Probably worth a comment in sym.cpp (and maybe even that comment added via this script, along the lines of "don't update the strings here manually, these are auto generated by ..."